.github/actions/promote-to-stable/action.yml

```diff
@@ -0,0 +1,91 @@
+name: "promote testing to stable"
+description: "Promote testing packages to stable."
+inputs:
+  artifactory_token:
+    description: "Artifactory token"
+    required: true
+  module:
+    description: "Module"
+    required: true
+  distrib:
+    description: "The distribution used for packaging"
+    required: true
+  major_version:
+    description: "Centreon packaged major version"
+    required: true
+  minor_version:
+    description: "Centreon package minor version"
+    required: true
+  stability:
+    description: "The package stability (stable, testing, unstable)"
+    required: true
+
+runs:
+  using: "composite"
+  steps:
+    - uses: jfrog/setup-jfrog-cli@v3
+      env:
+        JF_URL: https://centreon.jfrog.io
+        JF_ACCESS_TOKEN: ${{ inputs.artifactory_token }}
+
+    - name: Promote RPM packages to stable
+      if: ${{ startsWith(inputs.distrib, 'el') }}
+      run: |
+        set -x
+        echo "[DEBUG] - Major version: ${{ inputs.major_version }}"
+        echo "[DEBUG] - Minor version: ${{ inputs.minor_version }}"
+        echo "[DEBUG] - Distrib: ${{ inputs.distrib }}"
+
+        for ARCH in "noarch" "x86_64"; do
+          echo "[DEBUG] - Get path of $ARCH testing artifacts to promote to stable."
+          SRC_PATHS+=$(jf rt s --include-dirs rpm-standard/${{ inputs.major_version }}/${{ inputs.distrib }}/testing/$ARCH/${{ inputs.module }}/*.rpm | jq -r '.[].path')
```

Comment:
Shouldn't SRC_PATH be set before being affected to this operation ?

Reply:
Added initialization before first loop